doc/api/stream.md

```diff
@@ -2647,15 +2647,23 @@ const finished = util.promisify(stream.finished);
 
 const writable = fs.createWriteStream('./file');
 
-(async function() {
+async function pump(iterator, writable) {
   for await (const chunk of iterator) {
     // Handle backpressure on write().
+    if (writable.destroyed) break;
     if (!writable.write(chunk))
       await once(writable, 'drain');
+    if (writable.destroyed) break;
```

Comment:
Yea... this `writable.destroyed` checks are ugly but I don't see a way around it...